README.md

```diff
@@ -106,7 +108,7 @@ TOKEN_EOF(lexeme="", line=2, column=2)
 
 ### Syntactical Grammar
 
-The syntactical grammar _should_ and LALR(1) grammar, i.e. it can be parsed by a left-to-right parser with 1 token of look-ahead. In SolScript, a program Source is a series of Statements. Statements use Expressions and Literals. Expressions are evaluated to a Value at run time.
+The syntactical grammar _should_ be an LALR(2) grammar, i.e. it can be parsed by a left-to-right parser with 2 tokens of look-ahead. In SolScript, a program Source is a series of Statements. Statements use Expressions and Literals. Expressions are evaluated to a Value at run time.
```

Comment:
Ideally this should be LALR(1). The Dragon Book says any LALR(k) grammar can be converted to LALR(1).